src/VirtoCommerce.Platform.Core/PlatformOptions.cs

```diff
@@ -26,6 +25,7 @@ public class PlatformOptions
         //Default path to store export files 
         public string DefaultExportFolder = "App_Data/Export/";
         public string DefaultExportFileName = "exported_data.zip";
+        public string LibraryPath { get; set; }
```

Comment:
I think there is need default value. Otherwise it will fail in case of somebody forget to add line to appsettings.json